hoomd/mpcd/collide.py

```diff
@@ -1,367 +1,225 @@
-
-    Note:
-        The *period* must be chosen as a multiple of the MPCD
-        :py:mod:`~hoomd.mpcd.stream` period. Other values will
-        result in an error when ```hoomd.run``` is called.
-
-    When the total mean-free path of the MPCD particles is small, the underlying
-    MPCD cell list must be randomly shifted in order to ensure Galilean
-    invariance. Because the performance penalty from grid shifting is small,
-    shifting is enabled by default in all simulations. Disable it using
-    :py:meth:`set_params()` if you are sure that you do not want to use it.
-
-    HOOMD particles in *group* can be embedded into the collision step (see
-    ``embed()``). A separate integration method (:py:mod:`~hoomd.md.methods`)
-    must be specified in order to integrate the positions of particles in *group*.
-    The recommended integrator is :py:class:`~hoomd.md.methods.NVE`.
+    rule for MPCD as first proposed by `Malevanets and Kapral
+    <http://doi.org/10.1063/1.478857>`_. Every `every` steps, the particles are
+    binned into cells. The particle velocities are then rotated by `angle`
+    around an axis randomly drawn from the unit sphere. The rotation is done
+    relative to the average velocity, so this rotation rule conserves linear
+    momentum and kinetic energy within each cell.
 
     The SRD method naturally imparts the NVE ensemble to the system comprising
     the MPCD particles and *group*. Accordingly, the system must be properly
     initialized to the correct temperature. (SRD has an H theorem, and so
-    particles exchange momentum to reach an equilibrium temperature.) A thermostat
-    can be applied in conjunction with the SRD method through the *kT* parameter.
-    SRD employs a `Maxwell-Boltzmann thermostat <https://doi.org/10.1016/j.jcp.2009.09.024>`_
-    on the cell level, which generates the (correct) isothermal ensemble. The
-    temperature is defined relative to the cell-average velocity, and so can be
-    used to dissipate heat in nonequilibrium simulations. Under this thermostat, the
-    SRD algorithm still conserves momentum, but energy is of course no longer conserved.
-
-    Examples::
-
-        collide.srd(seed=42, period=1, angle=130.)
-        collide.srd(seed=77, period=50, angle=130., group=hoomd.group.all())
-        collide.srd(seed=1991, period=10, angle=90., kT=1.5)
+    particles exchange momentum to reach an equilibrium temperature.) A
+    thermostat can be applied in conjunction with the SRD method through the
+    `kT` parameter. SRD employs a `Maxwell-Boltzmann thermostat
+    <https://doi.org/10.1016/j.jcp.2009.09.024>`_ on the cell level, which
+    generates the (correct) isothermal ensemble. The temperature is defined
+    relative to the cell-average velocity, and so can be used to dissipate heat
+    in nonequilibrium simulations. Under this thermostat, the SRD algorithm
+    still conserves linear momentum, but kinetic energy is of course no longer
```

Comment:
```suggestion
    still conserves linear momentum, but kinetic energy is no longer
```
Try to avoid language like obviously, clearly, of course, etc.

Reply:
That is a copy–paste from the original documentation. All of docstrings need review and revision, and will keep an eye out for this.